### PR TITLE
Tie feature request to a plugin

### PR DESCRIPTION
**Pull request by richardtief**

### CODE REVIEW

**Review by auhlig — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 1]
```diff
@@ -1,9 +1,11 @@
-name: "💡 Feature Request"
-description: Create a ticket requesting a new feature
-title: "💡 [PLUGIN] - <title>"
+name: Feature Request
+description: Suggest an idea for this project
+title: "[name of the plugin e.g. kube-monitoring] Issue Title"
```
Comment (viennaa) [inline .github/ISSUE_TEMPLATE/feature-request.yml:3]:
```suggestion
title: "[name of the plugin] Issue Title"
```

would not give examples here, examples are the plugin names, which are in already

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 labels: [
   "plugin"
 ]
+assignees:
+  -
```
Comment (viennaa) [inline .github/ISSUE_TEMPLATE/feature-request.yml:8]: empty list here, should we maybe comment this out? 
or we some information who can be put in here?
actually this should be assigned by us, I'd say.

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 3]
```diff
@@ -20,14 +22,24 @@ body:
...
         - (Urgent) I can't use the Greenhouse without
+        
   - type: textarea
-    id: description
+    id: desc
     attributes:
-      label: "Description"
-      description: Provide a clear and concise description of the feature you're requesting
-      placeholder: Short and explicit description of your feature
+      label: Is your feature request related to a problem ?
```
Comment (viennaa) [inline .github/ISSUE_TEMPLATE/feature-request.yml:29]:
```suggestion
      label: Is your feature request related to a problem?
```

File: .github/ISSUE_TEMPLATE/feature-request.yml [hunk 4]
```diff
@@ -20,14 +22,24 @@ body:
         - (Low) Something is a little off
         - (Medium) I'm annoyed but I'll live
         - (Urgent) I can't use the Greenhouse without
+        
   - type: textarea
-    id: description
+    id: desc
```
Comment (viennaa) [inline .github/ISSUE_TEMPLATE/feature-request.yml:27]: is this change wanted?

**Review by viennaa — CHANGES_REQUESTED** [review]
Thanks, some remarks inline.

File: .github/ISSUE_TEMPLATE/feature-request.yml (hunk 2 above)
Comment (richardtief) [reply .github/ISSUE_TEMPLATE/feature-request.yml:8]: This actually prevents you from assigning someone. This should always be done by us.

File: .github/ISSUE_TEMPLATE/feature-request.yml (hunk 4 above)
Comment (richardtief) [reply .github/ISSUE_TEMPLATE/feature-request.yml:27]: No.
```suggestion
    id: description
```

**Review by viennaa — APPROVED** [review]
And go

**Pull request merged**